### DISCUSSION TITLE
DoS attacking my own server as a test

### DISCUSSION THREAD

**Discussion by jbmorris289** [SoftEtherVPN/SoftEtherVPN#1577] [General]
Hi,

I'm trying to run a DoS attack on my server as a test for a detection script I'm trying to make. I want to try to get the server to log "A DoS attack has been detected..."
The one idea that has come up during search so far is using DoS software such as [LOIC (Low Orbit Ion Cannon)](https://sourceforge.net/projects/loic/). I've tried to use that (with light settings) to no avail.

How would I go about this?
Server version reports as "5.01 Build 9672."

**Comment by chipitsine** [comment]
DoS attack detection is probably part of paid version. Not sure we have
this in open source.

What DoS detection you are talking about? Please share details

**Reply by jbmorris289** [reply]
If you asking about the script I'm making... it's basically a Fail2Ban like script I'm trying to write so that I can protect my server better, since I can't watch the logs all the time. I want it to be able to automatically blacklist bad acting IPs.

A while back, my server was the victim of the attack described in issue #1001, and I wasn't aware for a little while. My server was fine, but I don't want to that to happen again without being alerted to it at the very least, so I need a way to harden things up a bit.

If you're talking about in SoftEther... well, I've encountered a few instances of this message in my logs before (on an earlier version):
> A DoS attack on the TCP Listener (port 443) has been detected. The connecting source IP address is [IP] port number is [PORT]. This connection will be forcefully disconnected now.
I want to be able to automatically blacklist them too.

Note: I realized I wrote another comment and totally missed the reply box here, so this is reposted. Apologies for any extra notifications!
